ckanext/canada/templates/help.html

```diff
@@ -0,0 +1,13 @@
+{% extends "page.html" %}
+
+{% block primary %}
+  {% if g.is_registry %}
+    {# Registry #}
+    <h1>{{ _('Frequently Asked Questions (FAQ)') }}</h1>
+    {% block primary_content %}{% endblock primary_content %}
+    {{ faq_html|safe }}
+  {% else %}
+    {# Portal #}
+    {{ super() }}
+  {% endif %}
+{% endblock %}
```

Comment:
Just curious, does this kind of construct work? If so it's a little less complicated.
```suggestion
{% if g.is_registry %}
  {% block primary %}
    <h1>{{ _('Frequently Asked Questions (FAQ)') }}</h1>
    {% block primary_content %}{% endblock primary_content %}
    {{ faq_html|safe }}
  {% endblock %}
{% endif %}
```

Reply:
sadly this does not work :(

https://stackoverflow.com/questions/22655031/jinja2-how-to-put-a-block-in-an-if-statement#answer-22667404 Confirmed the answer here.